src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+
```

Comment:
redundant empty line(remove them everywhere in code)

Reply:
One empty line is always put in the following cases:
**Before a block of code or a single-line comment**
"For" is a block of code
If I'm wrong, please correct me.
https://comaqa.gitbook.io/java-automation/stil-napisaniya-koda/propuski